apps/wallet/src/ui/app/redux/slices/app/appType.ts

```diff
@@ -14,3 +16,8 @@ export function getFromLocationSearch() {
     }
     return AppType.Fullscreen;
 }
+
+export function getIsAppFullScreen() {
```

Comment:
Is not really full screen so I think it might be better approaching this function the other way around, so this function could be `getIsAppViewPopup` or similar, and update the logic

Reply:
And I would rename all "full screen" appearances because is not really full screen